Dapper.Tests/Dapper.Tests.csproj

```diff
@@ -6,38 +6,36 @@
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
-    <TargetFrameworks>net452;netcoreapp1.0</TargetFrameworks>
+    <TargetFrameworks>net452;netcoreapp1.0;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.0</TargetFrameworks>

```

Comment:
Removed in bdebedea412a5144f962f86a4fcf7e6677870673 that'll immediately follow this PR.